src/_fade.scss

```diff
@@ -28,17 +39,29 @@
 	animation-fill-mode:forwards;
 }
 
+.anim-fade--out--short {
+	animation:fade--out $duration-s ease-out;
+	animation-iteration-count:1;
+	animation-fill-mode:forwards;
+}
+
+/*
+	transitions for js, in response to events and user interaction
+*/
 .anim-opacity--0 {
-    opacity:0;
+	opacity:0;
 }
 
 .anim-opacity--1 {
-    opacity:1;
+	opacity:1;
 }
 
-// for popover example
-.anim-opacity--short {
-    transition: opacity $duration-s ease-in-out;
+.anim-transition--opacity--s {
```

Comment:
This is a good example of why I prefer writing out `short` instead of `s`. I misread this class as "`opacity` is `small`". The `short` makes it clear that the variant is duration-related.